docs/docs/naming.md

```diff
@@ -26,11 +26,36 @@ $ dig search.marathon.mesos
 search.marathon.mesos.		60	IN	A	10.9.87.94
 ```
  
+## Container IP A Records
+
+The A records above point to the host IP of a given service. If container IPs are supported by Mesos and the executor of a test, Mesos-DNS will create another A record for each of the A records above by prefixing it with `_container` and returning the container IP:
```

Comment:
prefixing a label with an underscore `_container` is non-standard for A records, although rfc2181 seems to blow the doors wide open w/ respect to allowed characters (from the perspective of the server) and puts the onus on the clients to properly parse labels with previously-non-standard characters.

https://www.rfc-editor.org/rfc/rfc2181.txt (sec 11)

It may be worth noting this somewhere.